### ISSUE TITLE
NPM Release

### ISSUE THREAD

**Issue by curran** [PMSI-AlignAlytics/scrollgrid#6]
Are there any plans to release this module as a package on NPM?

**Comment by johnkiernander** [comment]
Yes I'd be happy to release it.  My intention is to finish documenting it, build a website for it and then I'll make more of an effort to make it accessible.  At the moment I only use it through bower but there's no reason it can't be on npm.
